pkg/fqdn/name_manager_gc.go

```diff
@@ -40,17 +42,16 @@ var (
 // deleted from the global DNSCache. Until then, each of these IPs is
 // inserted into the global cache as a synthetic DNS lookup.
 func (n *NameManager) GC(ctx context.Context) error {
-	var (
-		GCStart = time.Now()
+	GCStart := time.Now()
+
+	// activeConnections holds DNSName -> single IP entries that have been
+	// marked active by the CT GC. Since we expire in this controller, we
+	// give these entries 2 cycles of TTL to allow for timing mismatches
+	// with the CT GC.
+	activeConnectionsTTL := int(2 * DNSGCJobInterval.Seconds())
+	activeConnections := NewDNSCache(activeConnectionsTTL)
 
-		// activeConnections holds DNSName -> single IP entries that have been
-		// marked active by the CT GC. Since we expire in this controller, we
-		// give these entries 2 cycles of TTL to allow for timing mismatches
-		// with the CT GC.
-		activeConnectionsTTL = int(2 * DNSGCJobInterval.Seconds())
-		activeConnections    = NewDNSCache(activeConnectionsTTL)
-	)
```

Comment:
This part of the change looks unrelated to commit `fqdn: (cleanup) use sets.Set instead of maps`. It should probably be part of the previous commit (`fqdn: move GC, cleanup, restore functions to NameManager`).

Reply:
Ooh, good catch.